### DISCUSSION TITLE
Icons on Taskbar

### DISCUSSION THREAD

**Discussion by RBZS1947** [community/community#56951] [New to GitHub]
Whenever a program is open, its icon on the taskbar widens and writes the name of the program, taking up several precious spaces.
How do I discontinue this?

**Comment by queenofcorgis** [comment]
Hi @RBZS1947 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.
